test/multi-module.jl

```diff
@@ -0,0 +1,30 @@
+module MultiUse
+    using Test
+    using OOPMacro
+
+    @class MUParent begin
+        pfield::Int
+        pfun(self) = self.pfield
+    end
+
+    module SomeOtherModule
```

Comment:
It would be better if this module is used in the following test.

Reply:
Hi, I'm not sure I understand. This test shows that even though MUParent is defined again in another module, we still get the one defined in our module.

Reply:
I mean that you should also test (instantiate) SomeOtherModule.MuParent.

Reply:
ok, done